src/extends/room/meta.js

```diff
@@ -164,6 +164,12 @@ Room.isHallway = function (name) {
 }
 
 Room.isClaimable = function (name) {
+  if (Game.gcl.level >= GCL_NOVICE && Game.map.getRoomStatus(name).status !== 'normal') {
+    return false
+  }
+  if (Game.gcl.level < GCL_NOVICE && Game.map.getRoomStatus(name).status !== 'novice') {
```

Comment:
Did you mean the opposite? If we have >= GCL_NOVICE rooms and the room is in a novice area?